modules/table/src/testFixtures/java/org/apache/ignite/distributed/ItTxTestCluster.java

```diff
@@ -836,6 +845,210 @@ private static String extractConsistentId(ClusterService nodeService) {
         return nodeService.topologyService().localMember().name();
     }
 
+    /**
+     * If colocation is enabled given method creates zone raft listener for specific zone partition if it wasn't created previously and
+     * populate it with table raft processor. If colocation is disabled given method will create table raft listener.
+     */
+    // TODO https://issues.apache.org/jira/browse/IGNITE-24798 Simplify after switching main towards colocation.
+    private RaftGroupListener getOrCreateAndPopulateRaftGroupListener(
+            String assignment,
+            int zoneId,
+            int partId,
+            int tableId,
+            PartitionDataStorage partitionDataStorage,
+            StorageUpdateHandler storageUpdateHandler,
+            TxStatePartitionStorage txStateStorage,
+            SafeTimeValuesTracker safeTimeTracker,
+            PendingComparableValuesTracker<Long, Void> storageIndexTracker,
+            CatalogService catalogService,
+            SchemaRegistry schemaRegistry
+    ) {
+        if (enabledColocation()) {
+            ZonePartitionId zonePartitionId = new ZonePartitionId(zoneId, partId);
+
+            var nodeSpecificZonePartitionRaftGroupListeners = zonePartitionRaftGroupListeners.computeIfAbsent(assignment,
```

Comment:
Could you please replace `var` with explicit type here? It's very difficult to realize what the type is

Reply:
Done.